src/coloring/edgeColoring.cpp

```
@@ -32,10 +32,10 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #include <utility>
 #include <string>
 
-#include "cpp_common/identifiers.hpp"
 #include <boost/graph/edge_coloring.hpp>
 #include <boost/graph/graph_utility.hpp>
 
+#include "cpp_common/base_graph.hpp"
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)